bitnami/wavefront/Chart.yaml

```diff
@@ -30,4 +30,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-wavefront-proxy
   - https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes
   - https://github.com/wavefrontHQ/wavefront-proxy
-version: 1.3.1
```

Comment:
I'm not sure sure bumping the minor version is enough. By changing the container to scratch there are some features that will work differently. For example, those using the `.Values.command` section expecting a container with a shell will fail, the same with the custom liveness probes and other parts. Maybe, just because of that, we could consider bumping the major version just in case.